usaspending_api/api_docs/api_documentation/spending_explorer.md

```diff
@@ -0,0 +1,95 @@
+## [Spending Explorer](#spending-explorer)
+
+**Route:** `/api/v2/spending/`
+
+**Method:** `POST`
+
+This route sends a request to the backend to retrieve spending data information through various types and filters.
+
+### Request
+
+```
+{
+    "type": "agency",
+    "filters": {
+        "fy": "2017"
+    }
+}
+```
+**Query Parameters Description**
+
+**type** - `required` - a string that contains the type of spending to explore on.
+  `explorer types` - *budget_function*, *budget_subfunction*, *federal_account*, *program_activity*, *object_class*, *recipients*, *awards*, *agency*
```

Comment:
This is a roundabout way of saying these are the only accepted value. Let's just say that:

> **type** - `required` - a string that contains the type of spending to explore on. It must be one of these values: *budget_function*, *budget_subfunction*, *federal_account*, *program_activity*, *object_class*, *recipients*, *awards*, *agency*